tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "module": "commonjs",
     "target": "ES2020",
     "outDir": "out",
-    "lib": ["ES2020"],
+    "lib": ["ES2021"],
```

Comment:
Why did this change?

Reply:
To get access to the `String.replaceAll` function